<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpielfeldGui</class>
 <widget class="QMainWindow" name="SpielfeldGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/new/prefix2/cave.png</selectedon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_bedienWidgets" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>761</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Sueden">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Norden">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Osten">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Westen">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>W</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gold">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Goldvorrat:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_leben">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Leben:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_posX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_posY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>34</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(85, 0, 0);
image: url(:/new/prefix2/rahmen.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_gold_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Inventar:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item0">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>45</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>45</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_item2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>45</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_text" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 199, 199);</string>
     </property>
     <widget class="QLabel" name="label_bildRaum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(157, 157, 157);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_raumText">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_raumUntertext">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
